.github/workflows/RunScript_build_and_test_on_push.yml

```
@@ -5,18 +5,23 @@ on:
     branches-ignore:
       - main
     paths:
-      - 'Frends.PowerShell.RunScript/**'
+      - "Frends.PowerShell.RunScript/**"
   workflow_dispatch:
     paths:
```

actionlint (1.7.4)

expected "inputs" key for "workflow_dispatch" section but got "paths"

(syntax-check)